source/RevitLookup/Views/Dialogs/FamilySizeTableDialog.xaml.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright 2003-2024 by Autodesk, Inc.
+// 
+// Permission to use, copy, modify, and distribute this software in
+// object code form for any purpose and without fee is hereby granted,
+// provided that the above copyright notice appears in all copies and
+// that both that copyright notice and the limited warranty and
+// restricted rights notice below appear in all supporting
+// documentation.
+// 
+// AUTODESK PROVIDES THIS PROGRAM "AS IS" AND WITH ALL FAULTS.
+// AUTODESK SPECIFICALLY DISCLAIMS ANY IMPLIED WARRANTY OF
+// MERCHANTABILITY OR FITNESS FOR A PARTICULAR USE.  AUTODESK, INC.
+// DOES NOT WARRANT THAT THE OPERATION OF THE PROGRAM WILL BE
+// UNINTERRUPTED OR ERROR FREE.
+// 
+// Use, duplication, or disclosure by the U.S. Government is subject to
+// restrictions set forth in FAR 52.227-19 (Commercial Computer
+// Software - Restricted Rights) and DFAR 252.227-7013(c)(1)(ii)
+// (Rights in Technical Data and Computer Software), as applicable.
+
+using RevitLookup.ViewModels.Dialogs;
+using Wpf.Ui;
+
+namespace RevitLookup.Views.Dialogs;
+
+public sealed partial class FamilySizeTableDialog
+{
+    private readonly IServiceProvider _serviceProvider;
+
+    public FamilySizeTableDialog(IServiceProvider serviceProvider, FamilySizeTable table)
+    {
+        DataContext = new FamilySizeTableDialogViewModel(table);
+        _serviceProvider = serviceProvider;
+        InitializeComponent();
+    }
+    
+    public async Task ShowAsync()
+    {
+        var dialogOptions = new SimpleContentDialogCreateOptions
+        {
+            Title = "Family size table",
+            Content = this,
+            CloseButtonText = "Close",
+            DialogMaxWidth = 1500
```

Comment:
I guess it's better to remove the max width if we want to see large horizontal tables in the full screen mode